website/src/redux/reducers/ReducerConstants.jsx

```diff
@@ -24,29 +21,21 @@
     USER_EMAIL: null,
   },
   CANVASES: {},
-  CURRENT_CANVAS: {
-    CANVAS_NAME: null,
-    CANVAS_OWNER: null,
-    CANVAS_USERS: [],
-    CANVAS_ELEMENTS: {},
-    CANVAS_ACTIVE_ELEMENT: null,
-  }
+  CURRENT_CANVAS: null,
```

Comment:
There will be a list of canvases with names, users etc. and then current canvas will point to a specific one. This way we have the information cached if we come back to a canvas.